site/content/docs/5.3/examples/cards-rtl/index.html

```diff
@@ -19,6 +19,16 @@ <h1 class="visually-hidden">أمثلة على البطاقة</h1>
     </symbol>
   </svg>
 
+  <div class="alert alert-info mb-0" role="alert">
```

Comment:
TBH I'm not a big fan of this callout. We tend to avoid having callouts in example pages since it's not really the documentation. I still don't like too much the one in title bars.
Moreover, the difference between a button and a button-link is already explained in the docs. IMO we don't have to explain it everywhere everytime.
Thoughts here? Do you find it mandatory?

Reply:
IMHO, it might be a bit too much indeed since it seems to be basic web. 
I've set it because of a11y team revendications but I'd agree to remove it on my side. Then maybe it's a bit more tricky here, what would you think about inverting the favorite and share to buttons and the actual button to a link ? Might feel a bit better in here.

Note to myself: Not a big fan using `role="alert"` on a static element that don't need much attention.